installer/PowerToysSetupCustomActions/CustomAction.cpp

```diff
@@ -26,6 +29,24 @@ const DWORD USERNAME_LEN = UNLEN + 1; // User Name + '\0'
 static const wchar_t* POWERTOYS_EXE_COMPONENT = L"{A2C66D91-3485-4D00-B04D-91844E6B345B}";
 static const wchar_t* POWERTOYS_UPGRADE_CODE = L"{42B84BF7-5FBF-473B-9C8B-049DC16F7708}";
 
+struct WcaSink : spdlog::sinks::base_sink<std::mutex>
```

Comment:
I'm not following how is this used? Logger is initialized with custom sink, but logger is never used after ? Or I'm missing something ?

Reply:
`registry.h` methods are used both in powerpreview project and msi custom actions. Msi logging is done via `WcaLog` calls instead of writing to a file, so we initialize the `Logger` with it.